app/assets/stylesheets/sephora_style_guide/_base.scss

```diff
@@ -25,9 +25,5 @@
 @import 'components/sprite_icons';
 @import 'components/svg_icons';
 
-// Partials
-@import 'partials/product_card';
```

Comment:
This is to prevent double rendering of styles within the style guide (they are already included in `application.scss`). Will manually add to the main repo.